src/styles/audio-recorder.css

```diff
@@ -9,14 +9,22 @@
     display: flex;
     align-items: center;
 
-    padding: 12px 12px;
     transition: all 0.2s ease-in;
 }
 
 .audio-recorder-mic {
     cursor: pointer;
     height: 16px;
     color: black;
+    padding: 12px;
+}
+
+.audio-recorder .audio-recorder-mic{
```

Comment:
Just a formatting thing, could you please add a space before the bracket.

Reply:
Fixed.